uber/config.py

```diff
@@ -158,11 +158,14 @@ def _unrepr(d):
 for _opt, _val in BADGE_PRICES['attendee'].items():
     PRICE_BUMPS[EVENT_TIMEZONE.localize(datetime.strptime(_opt, '%Y-%m-%d'))] = _val
 
```

Comment:
One thing I'd forgotten to fix is the fact that our ``int_enums`` currently assume that we want the integers to be the keys and the description to be the value.  This makes sense for things like donation tiers and shirt sizes, but for prices we want it the other way around, especially since unlike those other examples we actively want there to be multiple items with the same price.

So we're adding a type of enum where we leave the mapping as description-to-price.